Ladybird/Tab.cpp

```diff
@@ -521,6 +521,14 @@ void Tab::location_edit_return_pressed()
     navigate(m_location_edit->text());
 }
 
+void Tab::open_file()
+{
+    m_open_file = QFileDialog::getOpenFileName(this, tr("Open file"), QDir::homePath(), tr("All Files (*.*)"));
```

Comment:
Since we don't reference this object anywhere else (?) in the class, this can probably be a local variable. `auto filename =` or similar.